### PR TITLE
refactor: use the instanceRecord that already exists in instanceAdmin

### PR DESCRIPTION
**Pull request by katelynsills**

### CODE REVIEW

File: packages/zoe/src/types.js [hunk 1]
```diff
@@ -51,9 +51,9 @@
  * @property {MakeInstance} makeInstance
  * @property {Offer} offer
  * @property {(Instance) => Object} getPublicFacet
- * @property {IssuerKeywordRecord} getIssuers
- * @property {BrandKeywordRecord} getBrands
- * @property {Object=} getTerms
+ * @property {(Instance) => IssuerKeywordRecord} getIssuers
+ * @property {(Instance) => BrandKeywordRecord} getBrands
+ * @property {() => Object} getTerms
```
Comment (erights) [inline packages/zoe/src/types.js]: I see from the implementation that `getTerms` has an `Instance` parameter as well, as indeed it must.

File: packages/zoe/src/zoeService/zoe.js [hunk 2]
```diff
@@ -168,11 +159,9 @@ function makeZoe(vatAdminSvc) {
...
                 [keyword]: brand,
               };
-              brandToPurse.init(brand, E(issuer).makeEmptyPurse());
-              instanceTable.update(instance, {
-                issuerKeywordRecord: instanceRecord.issuerKeywordRecord,
-                brandKeywordRecord: instanceRecord.brandKeywordRecord,
-              });
+              if (!brandToPurse.has(brand)) {
+                brandToPurse.init(brand, E(issuer).makeEmptyPurse());
+              }
```
Comment (Chris-Hibbert) [inline packages/zoe/src/zoeService/zoe.js:164]: Shouldn't the keywordRecords be updated when a new Brand is added?

**Review by Chris-Hibbert — APPROVED** [review]
Yes, it's fine to store the data with the instanceAdmin. It wasn't clear that that was the appropriate place.

File: packages/zoe/src/zoeService/zoe.js (hunk 2 above)
Comment (katelynsills) [reply packages/zoe/src/zoeService/zoe.js:164]: They are, a few lines above this.

File: packages/zoe/src/types.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/src/types.js]: Good point. Fixed

**Pull request merged**